### PR TITLE
Add blog for app development on openshift using ADB

### PR DESCRIPTION
**Pull request by praveenkumar**
This blog post is for app development and deployment on openshift using Atomic Developer bundle.

### CODE REVIEW

File: source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+published: true
+comments: true
+---
+
+[The Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) is a prepackaged development environment filled production-grade pre-configured tools which also include [OpenShift origin](https://github.com/openshift/origin). Using ADB app-developers can easily start building and developing their application on OpenShift platform.
+
+READMORE
+
+In this blogpost we are going to learn step by step to create an application on OpenShift platform and deploy. Before we proceed any further I highly recommend you to go through [prerequisites guide](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst). ADB provide a [custom Vagranfile](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile) to setup and provision OpenShift platform.
+
```
Comment (LalatenduMohanty) [inline source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md]: IMO we should start the steps from git clone ADB repo , then going to the directory where the Vagrantfile for OpenShift exists and then doing a `vagrant up`.

File: source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Step 7. Get your application route to access from browser. (**nodejs-example-sample-project.centos7-adb.10.1.2.7.xip.io**)
+
+```
+$ ./oc get routes
+NAME             HOST/PORT                                                       PATH      SERVICE          TERMINATION   LABELS
+nodejs-example   nodejs-example-sample-project.centos7-adb.10.1.2.7.xip.io            nodejs-example                 app=nodejs-example,template=nodejs-example
+```
+
+So now you have a sample app running on OpenShift platform. You can do lot of experiments using [client binary](https://docs.openshift.org/latest/cli_reference/index.html) or from the browser. Give it a try and let us know your feedback/concern/issues.
```
Comment (LalatenduMohanty) [inline source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md]: Should we add some text and screenshots for users who would be interested to use the webconsole. Or may we just point them to some where which tells about how to use webconsole.

File: source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md (hunk 1 above)
Comment (praveenkumar) [inline source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md]: I am assuming a developer already know about vagrant and only require a vagrant file also prerequisites guide have required step to start with a vagrant box.

**Comment by jberkus** [comment]
@LalatenduMohanty, @praveenkumar is this ready to go, or not?

**Comment by praveenkumar** [comment]
@jberkus yes until @LalatenduMohanty doesn't have any more suggestion/comments.

File: source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md (hunk 1 above)
Comment (LalatenduMohanty) [inline source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md]: @praveenkumar I do not disagree with you. But I think just adding two extra steps makes this blog applicable to more audience. Hence the suggestion.

**Comment by LalatenduMohanty** [comment]
@praveenkumar @jberkus LGTM

**Comment by jberkus** [comment]
@bproffitt this is good to go.  please push one blog post on monday, and the other one on tuesday.  thanks!

**Comment by bproffitt** [comment]
Blog posted

**Pull request merged**